ksql-cli/src/main/java/io/confluent/ksql/cli/console/CliConfig.java

```diff
@@ -27,6 +27,7 @@
 public class CliConfig extends AbstractConfig {
 
   public static final String WRAP_CONFIG = "WRAP";
+  public static final String COLUMN_WIDTH_CONFIG = "COLUMN-WIDTH";
```

Comment:
for what it's worth, postgres does:
```
postgres=# \pset columns 20
Target width is 20.
```
I think `COLUMN-WIDTH` is easier to understand, but just something to note.